CHANGELOG/CHANGELOG-1.6.md

```diff
@@ -0,0 +1,114 @@
+  * Add Port Number to Audit Logging. ([#3277](https://github.com/antrea-io/antrea/pull/3277), [@qiyueyao])
+- [Flow Visibility] Add Grafana Flow Collector as the new visualization tool for flow records.
+  * Add Grafana dashboards, Clickhouse data schema, deployment files, and doc. ([#3063](https://github.com/antrea-io/antrea/pull/3063) [#3525](https://github.com/antrea-io/antrea/pull/3525), [@heanlan] [@zyiou] [@dreamtalen])
+  * Add support for exporting flow records to ClickHouse from Flow Aggregator. ([#3196](https://github.com/antrea-io/antrea/pull/3196) [#3526](https://github.com/antrea-io/antrea/pull/3526), [@wsquan171] [@dreamtalen])
+  * Add ClickHouse monitor to ensure data retention for in-memory ClickHouse deployment. ([#3244](https://github.com/antrea-io/antrea/pull/3244) [#3498](https://github.com/antrea-io/antrea/pull/3498), [@yanjunz97])
+- [Multicast] Support IGMPv3 leave action. ([#3389](https://github.com/antrea-io/antrea/pull/3389), [@wenyingd])
+- [Windows] Add support for EndpointSlices on Windows Nodes. ([#3321](https://github.com/antrea-io/antrea/pull/3321), [@XinShuYang])
+- Add SKIP_CNI_BINARIES environment variable to support skipping installing specified CNI plugins. ([#3454](https://github.com/antrea-io/antrea/pull/3454), [@jainpulkit22])
+- Support antrea-agent UBI8 based image. ([#3273](https://github.com/antrea-io/antrea/pull/3273), [@ksamoray])
+- Add the following documentations:
+  * Add [documentation](https://github.com/antrea-io/antrea/blob/v1.6.0/docs/service-loadbalancer.md) for ServiceExternalIP feature and Service of type LoadBalancer. ([#3322](https://github.com/antrea-io/antrea/pull/3322), [@hty690])
+  * Add [documentation](https://github.com/antrea-io/antrea/blob/v1.6.0/docs/minikube.md) for deploying Antrea to Minikube cluster. ([#3391](https://github.com/antrea-io/antrea/pull/3391), [@jainpulkit22])
+  * Add [documentation](https://github.com/antrea-io/antrea/blob/v1.6.0/docs/multicluster/antctl.md) for `antctl` Multi-cluster commands. ([#3414](https://github.com/antrea-io/antrea/pull/3414), [@bangqipropel])
+  * Add [documentation](https://github.com/antrea-io/antrea/blob/v1.6.0/docs/antrea-ipam.md) for Multiple-VLAN support. ([#3507](https://github.com/antrea-io/antrea/pull/3507), [@gran-vmv])
+  * Add [upgrade guide](https://github.com/antrea-io/antrea/blob/v1.6.0/docs/multicluster/upgrade.md) for Multi-cluster. ([#3374](https://github.com/antrea-io/antrea/pull/3374), [@luolanzone])
+  * Add [a per-rule example](https://github.com/antrea-io/antrea/blob/v1.6.0/docs/feature-gates.md#networkpolicystats) for NetworkpolicyStats docs . ([#3356](https://github.com/antrea-io/antrea/pull/3356), [@ceclinux])
+
+### Changed
+
+- [Windows] Use uplink MAC as source MAC when transmitting packets to underlay network from Windows Nodes. Security-related configurations like "Forged transmits" in VMware vSphere don't need to be degraded. ([#3516](https://github.com/antrea-io/antrea/pull/3516), [@wenyingd])
```

Comment:
I am not sure what you mean by "degraded" here?

Reply:
I wanted to say securty level degraded but guess it was unclear, how about the following?
> MAC address spoofing configurations like "Forged transmits" in VMware vSphere don't need to be enabled"